docs/hub/how-to-inference.md

```diff
@@ -0,0 +1,44 @@
+# How to integrate the Inference API in your library
+
+The Inference API provides faster inference for your hosted models. Integration with the Hub allows you to access the Inference API programatically. This guide will show you how to make calls to the Inference API from your library. For more detailed information, refer to the [Inference API documentation](https://api-inference.huggingface.co/docs/python/html/index.html).
```

Comment:
Do we mean to say "fast inference" or is "faster" with reference to running inference on your local machine?

Reply:
Good point, I think it'd be safer to say "fast inference" just in case someone has some beastly machine at home 😂